activerecord/lib/active_record/database_configurations/hash_config.rb

```diff
@@ -91,6 +92,12 @@ def adapter
       def schema_cache_path
         configuration_hash[:schema_cache_path]
       end
+
+      def schema_dump
```

Comment:
I went with `schema_dump` to stay consistent with others boolean database configs:

```
prepared_statements: false
advisory_locks: false
```